src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt

```diff
@@ -129,17 +129,19 @@ class GitCommitAction : AnAction() {
             val selectionStart: Int
             if (useUnicode) {
                 var replaced = false
+                val textAfterUnicode = PropertiesComponent.getInstance(project).getValue(CONFIG_AFTER_UNICODE)
```

Comment:
```suggestion
                val textAfterUnicode = PropertiesComponent.getInstance(project).getValue(CONFIG_AFTER_UNICODE, " ")
```
Default value if the config has not be set.